domain/consensus/processes/consensusstatemanager/resolve_block_status.go

```diff
@@ -70,11 +71,12 @@ func (csm *consensusStateManager) resolveBlockStatus(stagingArea *model.StagingA
 			blockStatus = externalapi.StatusDisqualifiedFromChain
 		} else {
 			lastResolvedBlockUTXOSet = previousBlockUTXOSet
+			lastResolvedBlockHash = previousBlockHash
```

Comment:
Sorry for being annoying, but I realize now that oneBlockBeforeCurrentUTXOSet is actually a better name, but still not very clear.
Maybe we should call it oneBeforeLastResolvedBlockHash/UTXOSet?